grails-app/services/org/pih/warehouse/core/DocumentService.groovy

```diff
@@ -917,23 +917,17 @@ class DocumentService {
                 row.createCell(CELL_INDEX).setCellValue(itemInstance?.inventoryItem?.product?.name)
                 row.getCell(CELL_INDEX++).setCellStyle(tableDataLeftStyle)
 
-                String supplierOutput = ""
-                String manufacturerOutput = ""
-                String manufacturerCodeOutput = ""
+
                 def suppliersList = itemInstance?.orderItems*.productSupplier
                 ProductSupplier supplier = suppliersList?.get(0)
-                if (supplier && shipmentInstance.isFromPurchaseOrder) {
-                    supplierOutput += supplier.supplierCode
-                    manufacturerOutput += supplier.manufacturer
-                    manufacturerCodeOutput += supplier.manufacturerCode
-                }
-                row.createCell(CELL_INDEX).setCellValue(supplierOutput)
+
+                row.createCell(CELL_INDEX).setCellValue(supplier?.supplierCode ?: '')
                 row.getCell(CELL_INDEX++).setCellStyle(tableDataCenterStyle)
 
-                row.createCell(CELL_INDEX).setCellValue(manufacturerOutput)
+                row.createCell(CELL_INDEX).setCellValue(supplier?.manufacturer?.toString() ?: '')
```

Comment:
`toString()` was neccessary, because supplier.manufacturer is of `Organization` class and in this version it was said that 
```
setCellValue cannot be applied to '(java.io.Serializable & java.lang.Comparable<? extends java.io.Serializable & java.lang.Comparable<?>>)' 
```